src/App.js

```diff
@@ -2,28 +2,30 @@ import React from "react";
 import "./App.css";
 import Navbar from "./Components/Navbar";
 import New from './Components/New';
-import Past from  './Components/Past';
+import Job from  './Components/Job';
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import NavigationLinks from "./Components/NavigationLinks";
 import Footer from "./Components/Footer";
+import Best from "./Components/Best";
 
 function App() {
   return (
     <div className="App">
       <Router>
+        <Navbar />
+        <div className="navigation-links">
+          <NavigationLinks link="/" type="New" />
+          <NavigationLinks link="/job" type="Job" />
+          <NavigationLinks link="/best" type="Best" />
+        </div>
 
-      <Navbar />
-      <div className="navigation-links">
-        <NavigationLinks link="/" type="New"/>
-        <NavigationLinks link="/past" type="Past"/>
-      </div>
+        <Switch>
+          <Route exact path="/" component={New} />
+          <Route exact path="/best" component={Best} />
+          <Route exact path="/job" component={Job} />
```

Comment:
Instead of creating three different component, You can have single component which will listen to URL Query Params like
`/:type`
And based on this type you can fetch ids from the single component